mne/io/brainvision/brainvision.py

```diff
@@ -35,8 +35,8 @@ class RawBrainVision(BaseRaw):
     vhdr_fname : str
         Path to the EEG header file.
     montage : str | None | instance of Montage
-        Path or instance of montage containing electrode positions.
-        If None, sensor locations are (0,0,0). See the documentation of
+        Path or instance of montage containing electrode positions. If None,
+        read sensor locations if present. See the documentation of
```

Comment:
I hope I'm not rehashing an older discussion, but shouldn't this be more explicit, like "read sensor locations from .vhdr file if present, otherwise (0, 0, 0)"?

Reply:
yes the `else` / otherwise would be a good addition

Reply:
I can modify the doc if you want (or perhaps you just handle this internally?)